sparkle/package.json

```diff
@@ -79,7 +79,7 @@
   "dependencies": {
     "@emoji-mart/data": "^1.1.2",
     "@emoji-mart/react": "^1.1.1",
-    "@headlessui/react": "^1.7.17",
```

Comment:
Heads up: are we okay with two different versions of headless in `sparkle` and `front`?

Reply:
Ahah I guess we are not, I'll bump the front's version as well but it requires more work. FWIW it's okay-ish as it does not break any existing behavior.

Reply:
Can help with that if needed 👍🏼

Reply:
All yours!